app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
I find the indentation in this file to be weird and after doing some inspecting I found out that this is using Tabs instead of Spaces to indent, right?

Please reconfigure your IDE to use Spaces instead. At least for this project.